### PR TITLE
Feature api

### PR DESCRIPTION
**Pull request by Bard-Budist**
endpoint and in index add new route /stats to show all count of the
class

### CODE REVIEW

File: api/v1/app.py [hunk 1]
```diff
@@ -7,16 +7,17 @@
...
     storage.close()
 
+
 if __name__ == "__main__":
     app.run(
         host=os.getenv('HBNB_API_HOST'),
         port=os.getenv('HBNB_API_PORT'),
-        threaded=True)
-    
+        threaded=True)
```
Comment (jdrestre) [inline api/v1/app.py:23]: Don't have newline at the end of file

**Review by jdrestre — CHANGES_REQUESTED** [review]
Intersting tools itertools for iteration into 2 list. Thanks for show me.
Review newline at the end file `app.py`

File: api/v1/app.py (hunk 1 above)
Comment (Bard-Budist) [reply api/v1/app.py:23]: Pycodestyle must the file dont have new line at the end of file

**Review by jdrestre — APPROVED** [review]
Ok. Done!

**Pull request merged**